playbooks/ceph.yml

```diff
@@ -266,6 +266,7 @@
         tasks_from: pools
 
     - name: Deploy RGW
+      when: cifmw_ceph_daemons_layout.rgw_enabled | bool
```

Comment:
`cifmw_ceph_daemons_layout.rgw_enabled | default(false) | bool`

and same pattern for others

Reply:
Thank john, updated it in the latest patch.